CHANGELOG

```diff
@@ -1,6 +1,26 @@
 devel
 -----
 
+* Extend COLLECT optimizer rule to to consider existing SORT nodes that cover
+  all group variables. Previously the following query would use a hash collect
+  instead of a sorted collect:
```

Comment:
```suggestion
* Extend COLLECT optimizer rule to consider existing SORT nodes that cover
  all group variables. Previously the following query would use a hash collect
  instead of a sorted collect:
```